src/apis/vaults.ts

```diff
@@ -250,9 +252,14 @@ export class DefaultVaultsAPI {
         return new UInt(new TypeRegistry(), 0) as PolkaBTC;
     }
 
-    private async getSecureCollateralThreshold(): Promise<Big> {
-        const secureCollateralThresholdU128 = await this.api.query.vaultRegistry.secureCollateralThreshold();
-        return new Big(secureCollateralThresholdU128.toString());
+    async getAuctionCollateralThreshold(): Promise<Big> {
```

Comment:
can you also move them down to line 308 so we have it with the rest of the thresholds

also, can you transform the other thresholds to return a `Big` please?